data/data/bootstrap/files/etc/mdsd.d/mdsd.env.template

```diff
@@ -0,0 +1,15 @@
+MONITORING_GCS_ENVIRONMENT={{ .LoggingConfig.Environment }}
+MONITORING_GCS_ACCOUNT={{ .LoggingConfig.Namespace }}
+MONITORING_GCS_REGION={{ .LoggingConfig.Region }}
+MONITORING_GCS_CERT_CERTFILE=/etc/mdsd.d/secret/mdsdcert.pem
+MONITORING_GCS_CERT_KEYFILE=/etc/mdsd.d/secret/mdsdcert.pem
+MONITORING_GCS_NAMESPACE=AROClusterLogs
+MONITORING_CONFIG_VERSION={{ .LoggingConfig.ConfigVersion }}
+MONITORING_USE_GENEVA_CONFIG_SERVICE=true
+MONITORING_TENANT={{ .LoggingConfig.Region }}
```

Comment:
Is this right? Tenant = Region?

Reply:
Yes, it's the region written in short form "eastus".

Reply:
see https://github.com/Azure/ARO-RP/blob/master/pkg/genevalogging/genevalogging.go#L586 and https://github.com/Azure/ARO-RP/blob/master/pkg/genevalogging/genevalogging.go#L562